packages/style-engine/src/styles/utils.ts

```diff
@@ -46,3 +90,47 @@ export function generateBoxRules(
 
 	return rules;
 }
+
+/**
+ * Returns a slug from a style value following the pattern `var:preset|styleContext|slug`.
+ *
+ * @param  styleValue   A raw style value.
+ * @param  styleContext A style context to parse
+ *
+ * @return generated styles.
+ */
+export function getSlugFromPreset(
```

Comment:
Is this function being used anywhere yet? I couldn't see it being called from anywhere, so wondered if it was part of the `getClassnames` behaviour that was removed. Probably fine to leave it in, just curious!

Reply:
Another good call! 😄 

This was an artefact from testing returning css vars for elements. I removed it with the intention of looking at it in another PR, after the [backend](https://github.com/WordPress/gutenberg/pull/40987) is done